custom_components/tahoma/climate_devices/valve_heating_temperature_interface.py

```diff
@@ -0,0 +1,76 @@
+"""Support for ValveHeatingTemperatureInterface."""
```

Comment:
I think you forked from the wrong branch.

Reply:
Oops, I will fix this :-)